code/index.html

```diff
@@ -1,12 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
-  </head>
-  <body>
-    <h1>This is your Pizza Bot</h1>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Cabin&display=swap" rel="stylesheet">
+  <title>Project 4: Pizzabot</title>
+</head>
+
+<body>
+  <h1>Hello Pizza Lover!</h1>
+  <button onclick="buttonClicked()"> Order your pizza here!</button>
```

Comment:
A tip is the name the function after what it does. We now that you are clicking something because `onclick` but `buttonClicked()` doesn't say so much of what will happen. Maybe `beginOrder()` or something would do the trick?